docs/bqetl.md

```diff
@@ -0,0 +1,126 @@
+# bqetl CLI
+
+The `bqetl` command-line tool aims to simplify working with the bigquery-etl repository by supporting common workflows, such as creating, validating and scheduling queries or adding new UDFs.
+
+## Installation
+
+Please follow the [Quick Start](https://github.com/mozilla/bigquery-etl#quick-start) for setting up bigquery-etl and the bqetl CLI.
+
+## Commands
+
+To list all available commands in the bqetl CLI:
+
+```bash
+$ ./bqetl
+
+Usage: bqetl [OPTIONS] COMMAND [ARGS]...
+
+  CLI tools for working with bigquery-etl.
+
+Options:
+  --version  Show the version and exit.
+  --help     Show this message and exit.
+
+Commands:
+  alchemer    Commands for importing alchemer data.
+  dag         Commands for managing DAGs.
+  dependency  Build and use query dependency graphs.
+  dryrun      Dry run SQL.
+  format      Format SQL.
+  glam        Tools for GLAM ETL.
+  mozfun      Commands for managing mozfun routines.
+  query       Commands for managing queries.
+  routine     Commands for managing routines.
+  stripe      Commands for Stripe ETL.
+  view        Commands for managing views.
```

Comment:
I would probably skip showing the output, as it's just one more thing to keep up to date.

```suggestion
```

Reply:
This is a tough one. I agree there's danger of this getting out of date, but it's also a nice concise summary of what to expect, which could provide motivation for users to go ahead and invest a few minutes in getting the CLI installed to play around with it more.

Reply:
I think this is useful. Unfortunately, I couldn't find a way to get this information programmatically.

Reply:
Could you run `./bqetl` by itself in CI (as you do in the example), capture its output, then paste it in here?

Reply:
I think this could be possible. My concern would be that the documentation process it might get harder to understand (some things happen in CI, some as Python scripts, some parts are static).

I'll open a follow-up issue to investigate this further at some point in the future.